packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Description.tsx

```diff
@@ -0,0 +1,88 @@
+import { ENTER } from '@codesandbox/common/lib/utils/keycodes';
+import React, {
+  ChangeEvent,
+  FunctionComponent,
+  KeyboardEvent,
+  useState,
+} from 'react';
+import styled from 'styled-components';
+
+import { useOvermind } from 'app/overmind';
+
+import { Textarea, Stack } from '@codesandbox/components';
+import { PenIcon } from './icons';
+
+const Icon = styled(PenIcon)`
+  cursor: pointer;
+  display: none;
+`;
+
+const Name = styled(Stack)<{ empty: boolean }>`
```

Comment:
This variable name is a lie. Also sounds like you need 2 different elements?

One for the container and the other for font styles for description text

Reply:
I updated this yeah :)